.travis.yml

```diff
@@ -12,7 +12,8 @@ sudo: false
 install:
     - pip install Sphinx==1.3b1
     - pip install -r requirements-travis.txt
-    - git clone --depth 1 https://github.com/avocado-framework/avocado.git --branch 69lts avocado-libs
+    - if [[ $TRAVIS_PYTHON_VERSION == "2.7" ]]; then git clone --depth 1 https://github.com/avocado-framework/avocado.git --branch 69lts avocado-libs; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == "3."* ]]; then git clone --depth 1 https://github.com/avocado-framework/avocado.git avocado-libs; fi
```

Comment:
Well if they ever release python4 we'd end-up without any avocado. How about something like this?

```
git clone --depth 1 https://github.com/avocado-framework/avocado.git --branch $([ $TRAVIS_PYTHON_VERSION == "2.7" ] && echo 69lts || echo master) avocado-libs
```

Reply:
... or if/else/fi, but I like the echo solution better...

Reply:
@ldoktor Thanks, updated